pkg/eufy_adapter.py

```diff
@@ -63,16 +63,19 @@ def start_pairing(self, timeout):
                 model = dev['type']
                 name = dev['name']
 
-                if model in ['T1201', 'T1202', 'T1203', 'T1211']:
-                    eufy_dev = lakeside.switch(address, code, model)
-                    device = EufySwitch(self, _id, name, eufy_dev)
-                elif model in ['T1011', 'T1012', 'T1013']:
-                    eufy_dev = lakeside.bulb(address, code, model)
-                    device = EufyBulb(self, _id, name, eufy_dev)
+                try:
+                    if model in ['T1201', 'T1202', 'T1203', 'T1211']:
+                        eufy_dev = lakeside.switch(address, code, model)
+                        device = EufySwitch(self, _id, name, eufy_dev)
+                    elif model in ['T1011', 'T1012', 'T1013']:
+                        eufy_dev = lakeside.bulb(address, code, model)
+                        device = EufyBulb(self, _id, name, eufy_dev)
+                    else:
+                        continue
+                except OSError:
+                    pass
                 else:
-                    continue
-
-                self.handle_device_added(device)
+                    self.handle_device_added(device)
```

Comment:
While this does the right thing I think simplifying this from a `try/except/else` to a `try/except` would be good for readability. This is just my perspective as someone who doesn't program in python very much so feel free to leave it as-is.